stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -101,6 +102,16 @@ class Value():
     def __init__(self, typecode_or_type: str, *args: Any, lock: bool = ...) -> None: ...
 
 # ----- multiprocessing function stubs -----
+def active_children() -> List[Process]: ...
+def allow_connection_pickling() -> None: ...
```

Comment:
This is undocumented and apparently does nothing in 3.6. I suppose there's little harm in including it though.